dynamic-rapidpro-workspace-settings/utils.js

```diff
@@ -51,41 +48,62 @@ const getInputs = (core) => {
   return inputs;
 };
 
-const getFormattedFlows = flows => `module.exports = ${util.inspect(flows)};\n`;
-
-const isValidFlows = data => Object.keys(data).length !== 0 && data.constructor === Object;
+const isValidObject = data => Object.keys(data).length !== 0 && data.constructor === Object;
 
 const run = async (githubWorkspacePath, params, fs) => {
   try {
     if (!githubWorkspacePath) {
       throw new Error(`GITHUB_WORKSPACE not defined`);
     }
     const secrets = getInputs(params);
-    if (!isValidFlows(secrets.rp_flows)) {
-      throw new Error(`Invalid flows data`);
+    const flattenedSecrets = flattenObject(secrets);
```

Comment:
Do you need to  reference both `secrets` and `flattenedSecrets`? Would working with flattenedSecrets be sufficient?